runtime/rococo/constants/src/weights/block_weights.rs

```diff
@@ -47,16 +43,16 @@ parameter_types! {
 	/// Calculated by multiplying the *Average* with `1` and adding `0`.
 	///
 	/// Stats [NS]:
-	///   Min, Max: 3_063_393, 3_278_407
-	///   Average:  3_098_278
-	///   Median:   3_076_394
-	///   Std-Dev:  46158.3
+	///   Min, Max: 3_680_262, 4_938_714
+	///   Average:  3_745_959
+	///   Median:   3_725_848
+	///   Std-Dev:  126490.36
 	///
 	/// Percentiles [NS]:
-	///   99th: 3_266_514
-	///   95th: 3_205_264
-	///   75th: 3_101_379
-	pub const BlockExecutionWeight: Weight = 3_098_278 * WEIGHT_PER_NANOS;
+	///   99th: 3_855_334
+	///   95th: 3_815_093
+	///   75th: 3_762_125
+	pub const BlockExecutionWeight: Weight = 3_745_959 * WEIGHT_PER_NANOS;
```

Comment:
This is a 20% increase, thats not too good.  
I assume the CI benchmarking machine is a bit slower somehow.  

Do you think we can still merge this @shawntabrizi ?

Reply:
If you think it is slower, we should re-run it.

But if these are the result, then these are the results...

End of the day, this is an increase of 700 micro-seconds:

(700 microseconds) / (2 seconds) =
0.00035 = .035%

It doesn't matter?